bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/gemini/seqcomp/BaselineReorderTest.scala

```diff
@@ -0,0 +1,79 @@
+package edu.gemini.spModel.gemini.seqcomp
+
+import edu.gemini.spModel.config.ConfigBridge
+import edu.gemini.spModel.config.map.ConfigValMapInstances.IDENTITY_MAP
+import edu.gemini.spModel.config2.ItemKey
+import edu.gemini.spModel.gemini.calunit.CalUnitParams._
+import edu.gemini.spModel.gemini.flamingos2.Flamingos2
+import org.junit.Assert._
+import org.junit.{Before, Test}
+
+final class BaselineReorderTest extends GcalExecTest {
```

Comment:
There was an existing test using JUnit that was very similar so I just went with the flow.